modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.cpp

```diff
@@ -40,18 +40,24 @@ using namespace sofa::defaulttype;
 
 ////////////////////////////////////////////    FACTORY    //////////////////////////////////////////////
 int MechanicalMatrixMapperClass = core::RegisterObject("This component allows to map the stiffness (and mass) matrix through a mapping.")
+
+        .add< MechanicalMatrixMapper<Rigid3dTypes, Rigid3dTypes> >(true)
```

Comment:
I think we shouldn't anymore use explicit type in component instantiation. 
So only Rigid3Types.